QuickOSM/quick_osm_processing/advanced/decorate_output.py

```diff
@@ -0,0 +1,171 @@
+"""Decorate the layer as a QuickOSM output."""
+
+from processing.algs.qgis.QgisAlgorithm import QgisAlgorithm
+from qgis.core import (
+    Qgis,
+    QgsCategorizedSymbolRenderer,
+    QgsLayerMetadata,
+    QgsProcessingAlgorithm,
+    QgsProcessingLayerPostProcessorInterface,
+    QgsProcessingOutputVectorLayer,
+    QgsProcessingParameterVectorLayer,
+    QgsRendererCategory,
+    QgsSymbol,
+    QgsWkbTypes,
+)
+from qgis.PyQt.QtGui import QColor
+
+from QuickOSM.core import actions
+from QuickOSM.qgis_plugin_tools.tools.i18n import tr
+
+__copyright__ = 'Copyright 2021, 3Liz'
+__license__ = 'GPL version 3'
+__email__ = 'info@3liz.org'
+
+
+class DecorateLayerAlgorithm(QgisAlgorithm):
+    """Decorate the layer as a QuickOSM output."""
+
+    LAYER = 'LAYER'
+    OUTPUT_LAYER = 'OUTPUT_LAYER'
+
+    def __init__(self):
+        """Constructor"""
+        super().__init__()
+        self.feedback = None
+        self.layer = None
+
+    @staticmethod
+    def name() -> str:
+        """Return the name of the algorithm."""
+        return 'decoratelayer'
+
+    @staticmethod
+    def displayName() -> str:
+        """Return the display name of the algorithm."""
+        return tr('Decorate a layer from OSM')
+
+    @staticmethod
+    def group() -> str:
+        """Return the group of the algorithm."""
+        return tr('Advanced')
+
+    @staticmethod
+    def groupId() -> str:
+        """Return the id of the group."""
+        return 'advanced'
+
+    def shortHelpString(self) -> str:
+        """Return an helper for the algorithm."""
+        return tr('Decorate the layer as an QuickOSM output.')
+
+    def flags(self):
+        """Return the flags."""
+        return super().flags() | QgsProcessingAlgorithm.FlagHideFromToolbox
+
+    def add_parameters(self):
+        """Set up the parameters."""
+        param = QgsProcessingParameterVectorLayer(
+            self.LAYER, tr('Layer')
+        )
+        help_string = tr('Path where the file will be download.')
+        if Qgis.QGIS_VERSION_INT >= 31500:
```

Comment:
You can now remove all lines with `QGIS_VERSION_INT < 31600`, because it is now the minimum version. Some code cleaning.

CTRL+MAJ+F => `QGIS_VERSION_INT` in PyCharm ;-)